golang/cosmos/Makefile

```diff
@@ -28,7 +27,6 @@ VersionName := $(shell echo "$(NAME)" | sed -e 's/[^A-Za-z0-9]//g')
 ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=$(VersionName) \
 		-X github.com/cosmos/cosmos-sdk/version.AppName=agd \
 		-X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
-		-X github.com/cosmos/cosmos-sdk/version.Commit=$(GIT_COMMIT) \
```

Comment:
@JimLarson @michaelfig can we lose this build info? Including it makes every git commit require rebuilding the whole SDK image

Reply:
I'm -0 on losing this (slightly against)... having the commit available from `agd version --long` has definitely helped validators diagnose "this is not the binary you think you are running right now" kinds of problems.

I'd prefer you override with `GIT_COMMIT=unimportant bin/agd build` or `make build GIT_COMMIT=unimportant` if you just want faster builds for images that won't be published.